frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditCheckbox.tsx

```diff
@@ -37,8 +37,8 @@ type EditCheckboxKeys = typeof EDIT_CHECKBOX_FIELD_KEYS[number]
 type EditCheckboxInputs = Pick<CheckboxFieldBase, EditCheckboxKeys> & {
   fieldOptions: string
   ValidationOptions: {
-    customMin?: number | string
-    customMax?: number | string
+    customMin: string
```

Comment:
should we only allow `number` instead of string so we do not need to do all the string transformations?

Reply:
sure!

Reply:
[6744778](https://github.com/opengovsg/FormSG/pull/3702/commits/6744778bc42caf15ca8ddd6ffcf75568869db583)

Reply:
further fixed in [2b16210](https://github.com/opengovsg/FormSG/pull/3702/commits/2b16210db1245956dc3a3e31af1876b2131f0fd2)